sync_diff_inspector/diff_test.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2024 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"testing"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetSnapshot(t *testing.T) {
+	cases := []struct {
+		latestSnapshot     []string
+		snapshot           string
+		expected           string
+		latestSnapshotRows string
+		snapshotRows       string
+	}{
+		{
+			latestSnapshot: []string{},
+			snapshot:       "1",
+			expected:       "1",
+		},
+		{
+			latestSnapshot: []string{"2"},
+			snapshot:       "",
+			expected:       "2",
+		},
+		{
+			latestSnapshot: []string{"0"},
+			snapshot:       "3",
+			expected:       "3",
+		},
+		{
+			latestSnapshot: []string{"4"},
+			snapshot:       "0",
+			expected:       "0",
+		},
+		{
+			latestSnapshot: []string{"5"},
+			snapshot:       "6",
+			expected:       "5",
+		},
+		{
+			latestSnapshot: []string{"7"},
+			snapshot:       "6",
+			expected:       "6",
+		},
+		{
+			latestSnapshot:     []string{"2016-10-08 16:45:26"},
```

Comment:
the result of SHOW MASTER STATUS should be a TSO like 454832342613098496, it should not be a datetime nor unix timestamp like 1735047693.684 🤔

Reply:
address